bundles/binding/org.openhab.binding.lcn/src/main/java/org/openhab/binding/lcn/common/LcnLogger.java

```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright (c) 2010-2015, openHAB.org and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.lcn.common;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Global logger used by the LCN binding.
+ * 
+ * @author Tobias J�ttner
+ */
+public class LcnLogger {
```

Comment:
i would suggest to remove this class completely. It removes the value of having different loggers (which can be filtered through logback.xml)